streamer/src/quic.rs

```diff
@@ -44,8 +69,13 @@ pub(crate) fn configure_server(
         .collect();
     let cert_chain_pem = pem::encode_many(&cert_chain_pem_parts);
 
-    let mut server_config = ServerConfig::with_single_cert(cert_chain, priv_key)
+    let server_tls_config = rustls::ServerConfig::builder()
+        .with_safe_defaults()
+        .with_client_cert_verifier(SkipClientVerification::new())
+        .with_single_cert(cert_chain, priv_key)
```

Comment:
so we need this because the self-signed client cert is not in the trust chain?

Reply:
Yes, that's correct.

Reply:
The client is also skipping the server certificate verification. Maybe we should look if there is some additional verification needed/possible on both sides.